src/components/ConnectionsTable/ConnectionList/ConnectionList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from "react";
+import styles from "@components/ConnectionsTable/ConnectionList/ConnectionList.module.scss";
+import { ConnectionItem } from "@components/ConnectionsTable/ConnectionItem/ConnectionItem";
+import { Connection } from "@models/Connection";
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "store";
+import {
+  BoardConnectionsMock,
+  ConnectionsMock,
+} from "@components/ConnectionsTable/structs/ConnectionsMock";
+import { initializeMockConnections } from "@slices/connectionsSlice";
+
+interface Props {
+  title: string;
+  connections: Connection[];
+  enabled?: boolean;
+}
+export const ConnectionList = ({
+  title,
+  connections,
+  enabled = true,
+}: Props) => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.title}>{title}</div>
+      <ul className={styles.connectionsList}>
+        {connections.map((item, index) => {
+          return (
+            <ConnectionItem key={index} connection={item} enabled={enabled} />
+          );
+        })}
+      </ul>
+    </div>
+  );
+};
+
```

Comment:
What do we do with the mocks? maybe it would be good not to delete them and leave them in a file in case they are needed later. Anyway, now I need them for the changes I have to make, so don't delete them.